docs/source/configuration.rst

```diff
@@ -1587,3 +1587,9 @@ This is the URL to which an XHR GET request will be made to fetch user data from
 The query string will be included in the request with ``q`` as its key.
 
 The data returned must be a JSON encoded list of user JIDs.
+
```

Comment:
Please mention here that the `?` is required. You can use the `.. note::` directive.